docs/develop/api-reference/index.mdx

```diff
@@ -196,6 +196,96 @@ puts client.caches.to_a.join(", ")
   cli={`momento cache list`}
 />
 
+### Flush cache
+
+Flushes all data from a cache
+
+Attributes:
+
+| Name      | Type   | Description                      |
+| --------- | ------ | -------------------------------- |
+| cacheName | String | Name of the cache to be flushed. |
+
+:::tip
+
+While you could use [Delete Cache](#delete-cache), then [Create Cache](#create-cache) to mimic this, the FlushCache API keeps the settings and only deletes the data in the cache.
```

Comment:
is this worth mentioning?

Reply:
IMO yes. Why else did we create the feature unless it was different enough from delete and then create?